spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -35,11 +38,12 @@ A full list of changes delivered in the 2.0 release can be found at link:https:/
 
 - OSGi dependencies marked as "provided" (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/132[#132])
 
-=== API/SPI Changes 
+=== API/SPI Changes
 - ReactiveStreamsFactory.fromSubscriber parameter type change (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/134[#134])
 
 === Other Changes
 - Update to Jakarta EE8 APIs for MP 4.0 (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/128[#128])
```

Comment:
This section should not be deleted.

Reply:
This is not deleted :-) The '-' is part of the text to make a list